### DISCUSSION TITLE
Lightweight mode like inVSCode

### DISCUSSION THREAD

**Discussion by SteveWolligandt** [mfussenegger/nvim-jdtls#492] [Q&A]
Hi everyone!

I am wondering if there is something like the lightweight mode in VSCode:

https://code.visualstudio.com/docs/java/java-project#_lightweight-mode

The problem is that a project spawns an error when I open a java file with neovim. 
However a colleague managed to make it work in VSCode with its lightweight mode for java. With this it ignores all of the gradle stuff and just takes the precompiled .classpath. Code navigation is working fine with this.

Thanks in advance,
Steve

**Comment by mfussenegger** [comment]
Are you referring to https://github.com/eclipse/eclipse.jdt.ls/wiki/Running-the-lightweight-syntax-language-server ?

You should be able to use that too, although you won't benefit from the extensions in nvim-jdtls.
